arroyo/backends/kafka/consumer.py

```diff
@@ -674,7 +677,14 @@ def __delivery_callback(
         payload: KafkaPayload,
         error: KafkaError,
         message: ConfluentMessage,
+        start_produce_time: float,
     ) -> None:
+        self.__metrics.timing(
+            "arroyo.producer.produce_time",
+            time.time() - start_produce_time,
+            tags={"topic": message.topic()},
+        )
+
```

Comment:
WE should debounce this. Producing one metric per message will be a problem in talking to veneur it would impact the throughput of the consumer too much.